### PR TITLE
[HV] Hibernate Validator 6.0.0.CR2 release

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: posts/Guillaume/2017-07-05-hibernate-validator-600-cr2-out.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+= "Hibernate Validator 6.0.0.CR2 is out with Bean Validation 2.0.0.CR2 support"
+Guillaume Smet
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+---
+We just released a CR2 of the Bean Validation 2.0 release train (e.g. the specification, the API and the TCK) and, as usual, we release a compatible version of Hibernate Validator shortly after: here comes Hibernate Validator 6.0.0.CR2.
```
Comment (gunnarmorling) [inline posts/Guillaume/2017-07-05-hibernate-validator-600-cr2-out.adoc:6]: Let's add a link once we got a post on bv.org.

File: posts/Guillaume/2017-07-05-hibernate-validator-600-cr2-out.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+== What's new since CR1
+
+As the final release of Bean Validation is getting closer, we mainly focused on keeping up with the latest spec clarifications and on updating the documentation.
+
+Here are the highlights of this small release:
+
+ * The `from` of `@ConvertGroup` and `<convert-group>` is now optional: it defaults to the `Default` group.
+ * Some last minute changes were made in the Bean Validation API on the naming of the value unwrapping related method: we updated Hibernate Validator accordingly.
+ * We made a first pass on the documentation: things are still missing (mostly about the new value extraction feature) but the documentation should not contain outdated information anymore.
```
Comment (gunnarmorling) [inline posts/Guillaume/2017-07-05-hibernate-validator-600-cr2-out.adoc]: s/things/some things/

File: posts/Guillaume/2017-07-05-hibernate-validator-600-cr2-out.adoc (hunk 1 above)
Comment (gunnarmorling) [reply posts/Guillaume/2017-07-05-hibernate-validator-600-cr2-out.adoc:6]: http://beanvalidation.org/news/2017/07/05/bean-validation-2-0-cr2-is-out/

**Comment by gsmet** [comment]
Merged, thanks.

**Pull request closed without merging**